webpack/assets/javascripts/react_app/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import URI from 'urijs';
+import PropTypes from 'prop-types';
+import AutoComplete from '../AutoCompleteSearch';
+import SearchButton from '../SearchButton';
+import Bookmarks from '../bookmarks';
+
+const handleSearch = (searchQuery = '') => {
+  const uri = new URI(window.location.href);
+  uri.setSearch('search', searchQuery.trim());
+  window.Turbolinks.visit(uri.toString());
```

Comment:
@ripcurld should we reuse updateTable ? (just wondering about the table /pagination usage cases)

Reply:
1) We can use `updateTable` here.
2) However, when `Table` or `Pagination` moves to React, `SearchBar` will have to be updated.
3) Nevertheless, either way `handleSearch` will have to be updated to a Redux action.

In conclusion, yes, we should call `updateTable` here.